### ISSUE TITLE
SP2019, ClientSidePage.Load("Home.aspx") - empty ClientSidePage

### ISSUE THREAD

**Issue by rychlym** [pnp/PnP-Sites-Core#2691]
Thank you for reporting an issue or suggesting an enhancement. We appreciate your feedback - to help the team to understand your needs, please complete the below template to ensure we have the necessary details to assist you. If you have a actual question, we would ask you to use [SharePoint Developer Group](http://aka.ms/sppnp-community) at Microsoft Tech Community. Thanks!

#### Which PnP repository should you use to report the issue? 

- PnP Sites Core - https://github.com/OfficeDev/PnP-Sites-Core

#### Category
[x ] Bug
[ ] Enhancement

#### Environment
[x] SharePoint 2019

#### Expected or Desired Behavior
var homePage = ClientSidePage.Load(ctx, "Home.aspx");
homePage.Controls.Count = 2 
There should be by default "News" and "Site Activity" web parts presented

#### Observed Behavior
After creating a new site on SP2019, ClientSidePage.Load method does not seem to load the structure.
homePage.Controls.Count = 0

![image](https://user-images.githubusercontent.com/11144487/85993590-d7182a80-b9f6-11ea-8569-ba954f0b6b0b.png)

It seems start to work since adding additional web-part to the page (and publishing the page).
![image](https://user-images.githubusercontent.com/11144487/85993153-2ad64400-b9f6-11ea-9ec8-0d5f011bba26.png)
(the image is showing situation after the home page copied (manually in the UI) to the Home1.aspx and added additional web-part.

#### Steps to Reproduce
Create a new modern Team site on the SP2019.  run the code above. Add e.g. a list web part at the beggining of the page and run again the above code to see the changes... 

Q:
Is there any quick programatical workaround (as e.g. adding a specific item property to the Home.aspx item?)

**Comment by rychlym** [comment]
Now I can think, that it is done by design -  done a kind of "initial easier" way (because e.g. the CanvasContent1 property  is empty - so there is nothing to be parsed )  for the just created modern home page. So it would be good to enhance the Load method for that initial case or  introduce a method to get the default page object model...

**Comment by peni4142** [comment]
Yeah, then it is bad design, Sometimes you don't know which run it is. Maybe you want to add something but in that case, you delete everything before.
